analysis/aggregate/name-aggregate-nodes.js

```diff
@@ -20,7 +21,29 @@ function getAggregateName (aggregateNode, sysInfo) {
   const interesting = toArray(frames.filter(frame => name(frame)))
 
   const userland = interesting.filter(isUserland(sysInfo))
-  if (userland.length) return name(userland[0])
+
+  // here is a list of observed generic identifiers where the lcoation should be better described, either by another
+  // frame in the node, or by the filename - this can be added to
+  const tooGeneric = [
+    'module.exports',
+    'handler',
+    'async.series',
```

Comment:
Maybe also `Promise` and `Promise.then` (though I'm not sure why I haven't seen either of those in any real samples - maybe because real instances of Promise are almost always given a name?)

Reply:
I found the instance of `promise` in the initial example was covered by the reducer that gets a more meaningful name from the later options in the userland unique names array (see [line 35](https://github.com/nearform/node-clinic-bubbleprof/pull/313/files#diff-272dbf608f87520f52bc7a9b80aa71d6R35))

Reply:
I actually initially had `Object` and removed it, because I wasn't sure if people would find the type names (`nextTick` etc) more useful in the context. I'll add it back in.

Reply:
In a lot of samples I've looked at, "Object" is replaced by the filename or if index.js directory, which is much more useful. E.g. on samples from AcmeAir it replaces "Object" with "routes" which is much more helpful.